rpcutils/rfq_marshal.go

```diff
@@ -1,14 +1,17 @@
-package rfqrpc
+package rpcutils
 
 import (
 	"fmt"
 	"math/big"
 
 	"github.com/lightninglabs/taproot-assets/rfqmath"
+	"github.com/lightninglabs/taproot-assets/taprpc/rfqrpc"
 )
 
-// UnmarshalFixedPoint converts an RPC FixedPoint to a BigIntFixedPoint.
-func UnmarshalFixedPoint(fp *FixedPoint) (*rfqmath.BigIntFixedPoint, error) {
+// UnmarshalRfqFixedPoint converts an RPC FixedPoint to a BigIntFixedPoint.
+func UnmarshalRfqFixedPoint(fp *rfqrpc.FixedPoint) (*rfqmath.BigIntFixedPoint,
```

Comment:
Looking at my previously proposed rule "put these functions into the package that defines the non-RPC type" should perhaps be a soft rule or "rule of thumb". Don't think placing this in the `rfqmath` package makes that much sense. But IMO it definitely makes sense in the `rfq` package.